src/swagger-specs/destinations.yaml

```diff
@@ -3086,6 +4975,449 @@ components:
+            updatedClient: example_updated client
+            sandboxId: '<sandbox id>'
+            sandboxName: '<sandbox name>'
+            imsOrgId: destinationapi@AdobeOrg
+            name: Another sample source connection to the Profile Store
+            connectionSpec:
+              id: 8a9c3494-9708-43d7-ae3f-cda01e5030e1
+              version: '1.0'
+            typeInfo:
+              type: "ProfileFragments"
+              id: "ups"
+            state: enabled
+            version: '"0e0d95ae-0000-0200-0000-62a55cdd0000"'
+            etag: '"0e0d95ae-0000-0200-0000-62a55cdd0000"'
+            inheritedAttributes:
+              baseConnection:
+                id: be3e2d03-6134-4566-9865-c40e0df8b718
+                connectionSpec:
+                  id: 8a9c3494-9708-43d7-ae3f-cda01e5030e1
+                  version: '1.0'     
+
+    SourceConnectionsResponseUPS:
+      summary: Source connection to UPS response example
+      value:
+        items:
+          - id: 097e3d72-921b-47f5-9e99-a4487cdbd1bd
+            createdAt: 1660155681202
+            updatedAt: 1660155681587
+            createdBy: example@AdobeID
+            updatedBy: example@AdobeID
+            createdClient: example_client
+            updatedClient: example_updated client
+            sandboxId: '<sandbox id>'
+            sandboxName: '<sandbox name>'
+            imsOrgId: destinationapi@AdobeOrg
+            name: Sample source connection to the Profile Store
+            connectionSpec:
+              id: 8a9c3494-9708-43d7-ae3f-cda01e5030e1
+              version: '1.0'
+            typeInfo:
+              type: "ProfileFragments"
+              id: "ups"
+            state: enabled
+            version: '"0e0d95ae-0000-0200-0000-62a21cdd0000"'
+            etag: '"0e0d95ae-0000-0200-0000-62a21cdd0000"'
+            inheritedAttributes:
+              baseConnection:
+                id: be3e2d03-6134-4566-9865-c40e0df8b718
+                connectionSpec:
+                  id: 8a9c3494-9708-43d7-ae3f-cda01e5030e1
+                  version: '1.0'                     
+
+    ConnectionsResponse:
+      summary: Connections response example.
+      value:
+        items:
+          - id: 9d8159c6-9da6-4205-8507-d818d7ad7103
+            createdAt: 1654785125699
+            updatedAt: 1654791389756
+            createdBy: example@AdobeID
+            updatedBy: example@AdobeID
+            createdClient: example_client
+            updatedClient: example_updated client
+            sandboxId: '<sandbox id>'
+            sandboxName: '<sandbox name>'
+            imsOrgId: destinationapi@AdobeOrg
+            name: amazon-s3
+            connectionSpec:
+              id: 4890fc95-5a1f-4983-94bb-e060c08e3f81
+              version: '1.0'
+            auth:
+              specName: Access Key
+              params:
+                authorizedDate: "2022-08-03"
+                s3SecretKey: '<secret key>'
```

Comment:
For scrubbed values in other swaggers I think we still use the curly brackets w/ screaming snake case like we do on our ExL guides, e.g.: `{SECRET_KEY}`